### PR TITLE
Protect images

### PR DESCRIPTION
**Pull request by thedac**
Script to set the image property protected to True if the image is in
use and False if not. For use to protect images from being pruned by
simplestreams.

### CODE REVIEW

**Comment by thedac** [comment]
FWIW, master fails CI as well.
https://github.com/openstack-charmers/bot-control/pull/26

Manual flake8 runs are clean.

File: tools/lib/common/tools_common.py [hunk 1]
```diff
@@ -571,3 +571,46 @@ def check_glance_client(gl=None):
     logging.info('Glance client check: '
                  '{}'.format(_check is not None))
     return _check is not None
+
+
+def validate_config(conf):
```
Comment (ChrisMacNaughton) [inline tools/lib/common/tools_common.py]: `conf` is unused in here

**Comment by ryan-beisner** [comment]
Master tests are resolved.  Please rebase.

**Comment by ryan-beisner** [comment]
Thanks for the two cool new tools.  :wrench:

**Pull request merged**